includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -304,6 +305,16 @@ private function normalize_attributes( $attributes ) {
 					// No need to copy.
 					break;
 
+				case 'type':
+					/*
+					 * Omit the `type` attribute if its value is `text/html`. Popular embed providers such as YouTube
+					 * and Amazon use this non-standard attribute.
+					 */
```

Comment:
```suggestion
					/*
					 * Omit the `type` attribute if its value is `text/html`. Popular embed providers such as Amazon
					 * Kindle use this non-standard attribute, which is apparently a vestige from <object>.
					 */
```

Reply:
@westonruter Updates reasoning behind omitting `type=“text/html”` attribute as you suggested.